benchmarks/gems_master_model/benchmark_amoebanet_gems_master.py

```diff
@@ -0,0 +1,292 @@
+import torch
+import torch.distributed as dist
+import torchvision.transforms as transforms
+import torchvision
+import numpy as np
+import time
+import sys
+import math
+import logging
+from models import amoebanet
+from torchgems import parser
+from torchgems.mp_pipeline import model_generator
+from torchgems.gems_master import train_model_master
+import torchgems.comm as gems_comm
+
+parser_obj = parser.get_parser()
+args = parser_obj.parse_args()
+
+if args.verbose:
+    logging.basicConfig(level=logging.DEBUG)
+
+gems_comm.initialize_cuda()
+
+
+class Unbuffered(object):
+    def __init__(self, stream):
+        self.stream = stream
+
+    def write(self, data):
+        self.stream.write(data)
+        self.stream.flush()
+
+    def writelines(self, datas):
+        self.stream.writelines(datas)
+        self.stream.flush()
+
+    def __getattr__(self, attr):
+        return getattr(self.stream, attr)
+
+
+def init_processes(backend="tcp"):
```

Comment:
should be `backend="mpi"`

Reply:
done all changes related to "tcp"->"mpi"